bindings/yarp.i

```diff
@@ -519,6 +521,7 @@ typedef yarp::os::BufferedPort<Sound> BufferedPortSound;
 %inline
 %{
 typedef yarp::sig::VectorOf<double> Vector;
+typedef yarp::sig::VectorOf<int> VectorInt;
```

Comment:
I don't have a strong opinion here, but perhaps the name `VectorOfInt` would be better? What do other people think?

Reply:
I decided for `VectorInt` just because with images the choice was for, e.g., `ImageRgb` instead of  `ImageOfRgb`.